public/data/samadpls.json

```
@@ -0,0 +1,33 @@
+{
+  "name": "Abdul Samad",
+  "type": "personal",
+  "bio": "Python Developer | HTML & CSS | CHISEL",
+  "avatar": "https://avatars.githubusercontent.com/u/94792103?v=4",
+  "links": [
+    {
+      "name": "Follow my Open Source on GitHub",
```

This looks great! :tada: Did you omit something here perhaps?
```suggestion
      "name": "Follow my Open Source journey on GitHub",
```

---

😊 thanks